### ISSUE TITLE
Mixer feature request.

### ISSUE THREAD

**Issue by Shadowhenge** [mschack/VCV-Rack-Plugins#79]
Love the mixers, but would love to see CV input level control for groups, and some sort of attenuverter to balance too loud or too weak input signals for finer mixing so the faders don't have to be in extreme positions or change the level drastically with slight movements.  would like an I/O expansion module for master clock that adds more clock/reset outs and perhaps even provides actual trigger and adjustable gate outs and delays.  A built in clock divider to give more than 4 clock options would also be welcome.

Thanks for all your hard work!

**Comment by mschack** [comment]
Thanks for the ideas.  I will look into adding a gain input to the groups.  x2 or x4 good enough?  As for the clock I may be designing a new one with more options.  I am not sure I understand what you mean by "4 clock options".  You mean more clock channels?

**Comment by Shadowhenge** [comment]
Yeah, a standard CV input for each group's fader would be fine.  Nothing
fancy.  As for the clock I meant either an expansion that adds more
outputs for the existing clock dividers, or it adds 4 more clock dividers
to the right of the existing ones... An 8 division clock.  For now I have
been just Daisy chaining more than one master clock module, or adding a ton
of mults to break out the extra ports and create trigger and gate busses.
Even if nothing changes they're great, though, so don't think I'm
complaining, hehe.
